src/core/thread/mle_router.cpp

```diff
@@ -2375,7 +2391,7 @@ otError MleRouter::HandleChildUpdateResponse(const Message &aMessage, const Ip6:
     LeaderDataTlv leaderData;
     Child *child;
 
-    otLogInfoMle(GetInstance(), "Received Child Update Response from child");
+    LogMleMessage("Receive Child Update Response from child", aMessageInfo.GetPeerAddr());
```

Comment:
Suggestion: adding rloc?

Reply:
Source Address TLV may or may not be included in this message.  I can move the Source Address parsing up and handle both cases.

Reply:
I see...I was thinking to use the `Child` entry's rloc here, but here we don't have the `child` yet.

Reply:
How about moving the log after we do `FindChild` ... something like this (it's a bit messy but will give us more info in case we are receiving child update from an unknown child):
```
   ...
   Child * child

    // Find Child
    aMessageInfo.GetPeerAddr().ToExtAddress(macAddr);

    child = FindChild(macAddr);

    if (child != NULL)
    {
        LogMleMessage("Receive Child Update Response from child", aMessageInfo.GetPeerAddr(), child->GetRloc16());
    }
    else
    {
        LogMleMessage("Receive Child Update Response from uknown child", aMessageInfo.GetPeerAddr());   
    }
```

Reply:
It's more complicated than that since we also validate the child's RLOC16 if we included one.  Anyways, I updated the code to handle the different cases.